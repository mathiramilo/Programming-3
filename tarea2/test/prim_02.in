leer_grafo
5 no-dirigido
1: 2 p5 3 p3 4 p6 .
2: 3 p2 .
3: 4 p4 5 p7 .
4: .
5: .
0: .
prim
Fin