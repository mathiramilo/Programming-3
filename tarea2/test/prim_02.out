1>El grafo leído es:
Imprimiendo Grafo
Cantidad de vértices: 5
Cantidad de aristas: 6
1: (2,5)(3,3)(4,6).
2: (1,5)(3,2).
3: (1,3)(2,2)(4,4)(5,7).
4: (1,6)(3,4).
5: (3,7).
2>El costo del MST es: 16, y el MST:
Imprimiendo Grafo
Cantidad de vértices: 5
Cantidad de aristas: 4
1: (3,3).
2: (3,2).
3: (1,3)(2,2)(4,4)(5,7).
4: (3,4).
5: (3,7).
3>Fin.
